src/external_task_api_service.ts

```diff
@@ -129,6 +128,57 @@ export class ExternalTaskApiService implements IExternalTaskApi {
     this._publishExternalTaskFinishedMessage(externalTask, successNotificationPayload);
   }
 
+  private async _fetchOrWaitForExternalTasks<TPayloadType>(
+    topicName: string,
+    maxTasks: number,
+    longPollingTimeout: number): Promise<Array<ExternalTask<TPayloadType>>> {
+
+    return new Promise<Array<ExternalTask<TPayloadType>>>(async (resolve): Promise<void> => {
+
+      const tasks: Array<ExternalTask<TPayloadType>> =
+        await this._externalTaskRepository.fetchAvailableForProcessing<TPayloadType>(topicName, maxTasks);
+
+      const taskAreNotEmpty: boolean = tasks.length > 0;
+
+      if (taskAreNotEmpty) {
+        resolve(tasks);
+        return;
```

Comment:
A spacing before a return increases the readability

```suggestion

        return;
```